lib/oga/xml/parser.rll

```diff
@@ -245,7 +245,7 @@ string_body
   def reset
     @line = 1
 
-    @lexer.reset
+    @lexer.reset_native
   end
```

Comment:
I originally removed this method but that caused test failures locally, so I added it back.